src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -685,8 +687,9 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
         OrchestratorKind::Kubernetes => {
             if args.orchestrator_process_scratch_directory.is_some() {
                 bail!(
-                    "--orchestrator-process-scratch-directory is \
-                      not currently usable with the kubernetes orchestrator"
+                    "Use --orchestrator-kubernetes-ephemeral-volume-class instead of \
+                    --orchesatrator-process-scratch-directory with the \
+                    kubernetes-orchestrator"
```

Comment:
Honestly this error message is probably too helpful. Zero people will be adding these options by hand and it's just going to rot. I liked the old error message better. We also don't bother rejecting any of the other `--orchestrator-process` options when you've set `--orchestrator=kubernetes`, so we could plausibly just remove the whole conditional.